scripts/sshg

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
```

Comment:
```suggestion

 ### Standalone Galaxy ssh helper bash script with multiple configurable options and defaults.
based on the sshy function developed by @cat-bro for MacOS and adapted to Linux by @neoformit
```

Reply:
Any reason for not specifying the bash shell? I haven't tested using other shells (sh, zsh for example).